maven-core/src/main/java/org/apache/maven/execution/ProfileActivation.java

```diff
@@ -34,9 +34,13 @@
  */
 public class ProfileActivation
 {
-
-    private static class ActivationSettings
+    private enum ActivationSettings
     {
+        ACTIVATE_OPTIONAL( true, true ),
```

Comment:
Shouldn't it reather be `ACTIVATION_...`/`DEACTIVATION_...`?

Reply:
Check.